databases/mysql57/Portfile

```diff
@@ -161,15 +160,6 @@ if {$subport eq $name} {
         }
     }
 
-    variant openssl description {Enable OpenSSL support} {
-        depends_lib-append      path:lib/libssl.dylib:openssl
```

Comment:
Is tis no longer required ?

Reply:
Yes, the `path:lib/libssl.dylib:openssl` dependency is already specified by the port without this variant.